lib/null_logger.rb

```diff
@@ -2,7 +2,16 @@
 # Is used when logger is not defined
 class NullLogger
   # Possible log levels from ruby Logger::Severity class
-  LOG_LEVELS = %w( unknown fatal error warn info debug ).freeze
+  LOG_LEVELS = %w(
+    unknown fatal error warn info debug
```

Comment:
Maybe if it is not one line anymore, so it would be better to have the next formatting, as we have almost everything for `%w` and `%i` within karafka.
```
  unknown
  fatal
  warn
  ...
```